components/core/DataView.js

```diff
@@ -248,20 +221,44 @@ export default class DataView extends React.Component {
     window.removeEventListener("remote-data-deletion", this._handleDataDeletion);
     window.removeEventListener("remote-slate-object-remove", this._handleRemoteSlateObjectRemove);
     window.removeEventListener("remote-slate-object-add", this._handleRemoteSlateObjectAdd);
+    window.removeEventListener("scroll", this._handleCheckScroll);
```

Comment:
on line 214 you added the event listener for the function this._hanldeScroll and on this line you removed it for the function this._handleCheckScroll. Shouldn't these match? Or is this intentional?